blueflood-elasticsearch/src/main/resources/init-es.sh

```diff
@@ -4,13 +4,25 @@
 #  are required to properly tokenize the incoming data.
 # You can pass it a url if your elastic search doesn't reside on localhost:9092
 #WARNING: this script will destroy existing ES data and reset it to the proper init state
-
+# Example:
+#    Local ES: ./init-es.sh
+#    ES on OR: ./init-es.sh <url for OR:ES> <username> <password>
 ELASTICSEARCH_URL=${1:-"http://localhost:9200"}
+ES_USERNAME=${2:-}
+ES_PASSWD=${3:-}
```

Comment:
This one dude on SO was quite against getopts and linked this:
http://mywiki.wooledge.org/BashFAQ/035#getopts

3 isnt that many imo, and there expected to be empty for 50% of the use-case i.e. the default use case.
But ill play around with getopts a bit and see how it feels in this scenario, might make sense. 
Alternatively it might make sense to have 2 command line args, with the second being <username>:<password>. That also eliminates a step in this script.